src/printer/text_printer.h

```diff
@@ -594,7 +594,8 @@ class TextPrinter {
     } else if (node.defined() &&
                (node->IsInstance<relax::VarNode>() || node->IsInstance<relax::FunctionNode>() ||
                 node->IsInstance<relax::ShapeExprNode>() ||
-                node->IsInstance<relax::RuntimeDepShapeNode>())) {
+                node->IsInstance<relax::RuntimeDepShapeNode>() ||
+                node->IsInstance<relax::TupleNode>())) {
```

Comment:
Here we fix a bug in text printer. Previously this code
```
x = R.Tuple(())
print(x)
```
reports an error `Check failed: (can_dispatch(n)) is false: NodeFunctor calls un-registered function on type relax.expr.Tuple` because PrettyPrint dispatches `relax.Tuple` wrong. This line of code adds the correct dispatch.